ledger/testing/consensusRange.go

```diff
@@ -57,7 +57,9 @@ var consensusByNumber = []protocol.ConsensusVersion{
 	protocol.ConsensusV33, // 320 rounds
 	protocol.ConsensusV34, // AVM v7, stateproofs
 	protocol.ConsensusV35, // minor, double upgrade withe v34
-	protocol.ConsensusV36, // box storage
+	protocol.ConsensusV36, // AVM v8, box storage
+	protocol.ConsensusV37, // ECDSA pre-check
+	protocol.ConsensusV38, // AVM v9, stateproofs recoverability
```

Comment:
I believe now this is the placement?
```suggestion
	protocol.ConsensusV37, // AVM v9, ECDSA pre-check
	protocol.ConsensusV38, // stateproofs recoverability, AVM v9 extra features
```

Reply:
all features moved to v38